### PR TITLE
Delete guest user from rabbitmq

### PR DESCRIPTION
**Pull request by jbadiapa**
Security issue

### CODE REVIEW

**Review by paramite — COMMENTED** [review]
Question: Do we need the rabbitmq_management plugin?

Nit: So I've just noticed your inconsistent indentation. Somewhere you use 1 space, somewhere 2 spaces and somewhere even 3 spaces. Please be consistent and use 2 spaces as in the rest of the project.

File: roles/rabbitmq/tasks/main.yml [hunk 1]
```diff
@@ -12,6 +12,14 @@
    enabled: yes
   when: manage_services|default(false)
 
+- name: Delete guest user on rabbitmq 
+  rabbitmq_user:
+     user:  '{{ rabbitmq_default_user }}' 
+     state: absent 
+  notify:
+      - restart rabbitmq
```
Comment (paramite) [inline roles/rabbitmq/tasks/main.yml]: Do we need to restart rabbitmq? IMHO server should take that into account even without restarting right?

Comment (mrunge) [reply roles/rabbitmq/tasks/main.yml]: I don't think we need to restart rabbitmq

Comment (jbadiapa) [reply roles/rabbitmq/tasks/main.yml]: It's not needed.
It doesn't hurt either.
At least it is what we said on the documentation.

**Review by larsks — APPROVED** [review]

**Pull request merged**